tests/devices/unit_tests/test_zebra.py

```diff
@@ -1,8 +1,7 @@
-from unittest.mock import AsyncMock
+from unittest.mock import AsyncMock, MagicMock
 
 import pytest
 from bluesky.run_engine import RunEngine
-from mockito import mock, verify
```

Comment:
How many places do we still use mockito? Maybe we can get rid of it as a dependency. I like fewer dependencies.

Reply:
Yh, I agree, thought about doing it as part of this but ran out of time. Made https://github.com/DiamondLightSource/dodal/issues/670